### DISCUSSION TITLE
[Unstable] diving suit lockers dont show the diving suits inside

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#14222] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

suit lockers with suits in them dont show the suit inside
![image](https://github.com/FakeFishGames/Barotrauma/assets/55637865/5bdfdc1c-b734-4a94-bad6-e4c27d16d1ae)
these all have suits

### Reproduction steps

put a suit in a suit locker and notice the lack of visual suit

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

Other

### -

unstable v1.6.1

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/f2d41c3d2224abc253d980a9a284dbc0b43c5c0f, coming up in the next unstable build!

**Reply by LePiromano** [reply]
i love you

**Comment by LePiromano** [comment]
the fix works
